R/collectors.R

```diff
@@ -200,6 +200,7 @@ guess_parser <- function(x, locale = default_locale()) {
 #'
 #' @param levels Character vector providing set of allowed levels.
 #' @param ordered Is it an ordered factor?
+#' @param include_na Should `NA` be included as a factor level?
```

Comment:
The current implementation includes NA regardless if it occurs in the input, I assume you would want it to only be included if the input contains NA?